### PR TITLE
[Enhancement](k8s) Add k8s yaml demo

### PR DESCRIPTION
**Pull request by zddr**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.
k8s yml demo
## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run p0

**Comment by zddr** [comment]
run buildall

File: docker/runtime/k8s/backend.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: dbe
```
Comment (caiconghui) [inline docker/runtime/k8s/backend.yml:21]: is namespace required filed in metadata?

Comment (caiconghui) [inline docker/runtime/k8s/backend.yml:21]:
```suggestion
  name: doris-be-demo
```

File: docker/runtime/k8s/backend.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  labels:
+    app: dbe
```
Comment (caiconghui) [inline docker/runtime/k8s/backend.yml:23]:
```suggestion
    app: doris-be-demo
```

File: docker/runtime/k8s/backend.yml [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      name: be
+    - port: 8040
+      name: webserver
+    - port: 9050
+      name: heartbeat-port #This name should be fixed. Doris will get the port information through this name
+    - port: 8060
+      name: brpc
+  clusterIP: None
+  selector:
+    app: dbe
```
Comment (caiconghui) [inline docker/runtime/k8s/backend.yml:36]:
```suggestion
    - port: 9060
      name: be_thrift_port
    - port: 8040
      name: be_webserver_port
    - port: 9050
      name: be_heartbeat_port #This name should be fixed. Doris will get the port information through this name
    - port: 8060
      name: be_brpc_port
  clusterIP: None
  selector:
    app: doris-be-demo
```

File: docker/runtime/k8s/backend.yml [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  be.conf: |
+    PPROF_TMPDIR="$DORIS_HOME/log/"
+    sys_log_level = INFO
+
+    be_port = 9060
+    webserver_port = 8040
+    heartbeat_service_port = 9050
+    brpc_port = 8060
+
+    priority_networks = 172.16.0.0/24
```
Comment (caiconghui) [inline docker/runtime/k8s/backend.yml:105]:
```suggestion
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: doris-be-demo
  labels:
    app: doris-be-demo
spec:
  selector:
    matchLabels:
      app: doris-be-demo
  serviceName: doris-be-demo
  replicas: 3
  template:
    metadata:
      name: doris-be-demo
      labels:
        app: doris-be-demo
    spec:
      containers:
        - name: doris-be-demo
          #Need to change to real mirror information
          image: apache-doris:test
          imagePullPolicy: IfNotPresent
          env:
            #Specify the startup type as k8s to bypass some restrictions of the official image initialization script
            - name: BUILD_TYPE
              value: "k8s"
          ports:
            - containerPort: 9060
              name: be_thrift_port
            - containerPort: 8040
              name: be_webserver_port
            - containerPort: 9050
              name: be_heartbeat_port
            - containerPort: 8060
              name: be_brpc_port
          volumeMounts:
              #Mount the configuration file in the way of configmap
            - name: conf
              mountPath: /opt/apache-doris/be/conf
              #Ifnot mounted, when enable_profile, error will be reported when querying the data from jdbc catalog
              #Error message: error setting certificate verify locations: CAfile:/etc/pki/tls/certs/ca-bundle.crt CApath: none
            - name: sys
              mountPath: /etc/pki
              readOnly: true
      volumes:
        - name: conf
          configMap:
            name: be-conf
        - name: sys
          hostPath:
            path: /etc/pki
---
apiVersion: v1
kind: ConfigMap
metadata:
  name: be-conf
data:
  be.conf: |
    PPROF_TMPDIR="$DORIS_HOME/log/"
    sys_log_level = INFO

    be_port = 9060
    webserver_port = 8040
    heartbeat_service_port = 9050
    brpc_port = 8060

    priority_networks = 172.16.0.0/24
```

**Comment by caiconghui** [comment]
yaml file name could be uniforma like: doris_be.yaml doris_fe.yaml doris_cn.yaml

**Comment by caiconghui** [comment]
and three compoment for fe be cn show be related if we want to fast to relate them in k8s cluster, doris-fe-$cluster_name
doris-be-$cluster_name doris-cn-$cluster_name

File: docker/runtime/k8s/backend.yml (hunk 3 above)
Comment (zddr) [reply docker/runtime/k8s/backend.yml]: k8s only support -, not _

**Review by caiconghui — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
run p0

**Pull request merged**